tests/runner.py

```diff
@@ -364,13 +364,21 @@ def get_setting(self, key):
       return self.settings_mods[key]
     return Settings[key]
 
-  def set_setting(self, key, value):
+  def set_setting(self, key, value=1):
+    if value is None:
+      self.clear_setting(key)
     self.settings_mods[key] = value
 
+  def clear_setting(self, key):
+    self.settings_mods.pop(key, None)
```

Comment:
hmm isn't `settings_mods` a dict? what does pop do here?

Reply:
```
pop(...)
    D.pop(k[,d]) -> v, remove specified key and return the corresponding value.
    If key is not found, d is returned if given, otherwise KeyError is raised
```

Allows me to avoid having to do a check when a `del`